models/DataObject/ClassDefinition/Data/ManyToManyRelation.php

```diff
@@ -1046,4 +1046,12 @@ public function getDiffDataFromEditmode($data, $object = null, $params = [])
 
         return;
     }
+
+    /**
+     * @return bool
+     */
+    public function getEnableAdminAsyncLoad()
```

Comment:
@fashxp Do we really want that enabled by default without having a choice? 
I think it would make sense to put the changes on `ManyToManyObjectRelation` into a trait and re-use it here and maybe also for `ReverseManyToManyObjectRelation`.

Reply:
Since the frontend/JS part is there, I assume that's not intentional ... I'm going to fix that.